common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3004,6 +3004,12 @@
     ],
     "sqlState" : "2200E"
   },
+  "NULL_QUERY_STRING_EXECUTE_IMMEDIATE" : {
+    "message" : [
+      "SQLQuery string should not be null."
```

Comment:
perhaps something like: `Execute immediate requires non-null value for sqlString variable, but provided <varName> is null`